<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>552</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>生机电虚拟环境测试平台</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="lineWidth">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_Add"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionReset"/>
   </widget>
   <widget class="QMenu" name="menu_S">
    <property name="title">
     <string>仿真(&amp;S)</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>直接加载</string>
    </property>
    <addaction name="actionSJT_3"/>
    <addaction name="actionSJT_6"/>
    <addaction name="actionBarrett"/>
    <addaction name="actionHumanHand"/>
    <addaction name="actionAddCustomHand"/>
    <addaction name="separator"/>
    <addaction name="actionAddObjects"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>模式识别</string>
    </property>
    <addaction name="actionTraining"/>
    <addaction name="actionTesting"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_S"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="viewToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_Add">
   <property name="text">
    <string>添加...(&amp;A)</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>设置...</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>停止</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>启动</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开文件...(&amp;O)</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存场景...(&amp;S)</string>
   </property>
  </action>
  <action name="actionSJT_6">
   <property name="text">
    <string>SJT_6</string>
   </property>
  </action>
  <action name="actionSJT_3">
   <property name="text">
    <string>SJT_3</string>
   </property>
  </action>
  <action name="actionStrategy">
   <property name="text">
    <string>策略选择</string>
   </property>
  </action>
  <action name="actionBarrett">
   <property name="text">
    <string>Barrett</string>
   </property>
  </action>
  <action name="actionHumanHand">
   <property name="text">
    <string>HumanHand</string>
   </property>
  </action>
  <action name="actionSample1">
   <property name="text">
    <string>Sample1</string>
   </property>
  </action>
  <action name="actionSample2">
   <property name="text">
    <string>Sample2</string>
   </property>
  </action>
  <action name="actionAddObjects">
   <property name="text">
    <string>添加抓取物体...</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>重置环境</string>
   </property>
  </action>
  <action name="actionAddCustomHand">
   <property name="text">
    <string>添加自定义手...</string>
   </property>
  </action>
  <action name="actionTraining">
   <property name="text">
    <string>Training</string>
   </property>
  </action>
  <action name="actionTesting">
   <property name="text">
    <string>Testing</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
